libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectUtils.kt

```diff
@@ -1,36 +1,39 @@
-@file:JvmName("RouteAlertUtils")
+@file:JvmName("RoadObjectUtils")
 
-package com.mapbox.navigation.base.trip.model.alert
+package com.mapbox.navigation.base.trip.model.roadobject
 
 import androidx.annotation.IntRange
 import com.mapbox.api.directions.v5.models.DirectionsRoute
 import com.mapbox.api.directions.v5.models.RouteOptions
 import com.mapbox.core.constants.Constants
 import com.mapbox.geojson.LineString
+import com.mapbox.navigation.base.utils.ifNonNull
 
 /**
- * Returns a [LineString] geometry of the alert.
+ * Returns a [LineString] geometry of the road object.
  *
- * @param directionsRoute the original route that this alert is a part of.
+ * @param directionsRoute the original route that this object is a part of.
  * @param precision precision of the polyline encoding.
  * Possible values are [Constants.PRECISION_5] and [Constants.PRECISION_6].
  * See [RouteOptions.Builder.geometries].
  */
-fun RouteAlertGeometry.toLineString(
+fun RoadObjectGeometry.toLineString(
     directionsRoute: DirectionsRoute,
     @IntRange(from = 5, to = 6) precision: Int
-): LineString {
+): LineString? {
     val routeLineString = LineString.fromPolyline(directionsRoute.geometry()!!, precision)
```

Comment:
Should we check the `precision` range and throw an `IllegalArgumentException` if not `5` or `6`? 🤔 

`IntRange` is just an annotation for the IDE but nothing prevents you from passing whatever you want at runtime.

Reply:
done